src/main/java/command/ListCommand.java

```diff
@@ -0,0 +1,17 @@
+package command;
+
+import cheatsheet.CheatSheetList;
+import parser.Parser;
+
+public class ListCommand extends Command {
+    public ListCommand(Parser parser) {
+        super(parser);
+    }
+
+    // todo: need to ask adhy whether he can attach index to cheatsheet
```

Comment:
hmm for this one I think just do it similar to our IP? create a variable inside a for loop in print method then just print the variable?